Corona-Warn-App/src/main/res/drawable-night/ic_dcc_reissuance_success_illustration.xml

```diff
@@ -0,0 +1,155 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:aapt="http://schemas.android.com/aapt"
+    android:width="360dp"
+    android:height="192dp"
+    android:viewportWidth="360"
+    android:viewportHeight="192">
+  <group>
+    <clip-path android:pathData="M0,0h360v192h-360z" />
+    <path android:pathData="M0,-56H360V192H0V-56Z">
+      <aapt:attr name="android:fillColor">
```

Comment:
Um 😐,this will cause a crash on `Android 6`

`android:fillColor`
Specifies the color used to fill the path. May be a color or, for SDK 24+, a color state list or a gradient color (See [R.styleable.GradientColor](https://developer.android.com/reference/android/R.styleable#GradientColor) and [R.styleable.GradientColorItem](https://developer.android.com/reference/android/R.styleable#GradientColorItem)). If this property is animated, any value set by the animation will override the original value. No path fill is drawn if this property is not specified.

Reply:
New illustrations are pushed.